.github/workflows/publish-commit-bottles.yml

```diff
@@ -63,47 +43,19 @@ jobs:
       - name: Push commits
         uses: Homebrew/actions/git-try-push@master
       - name: Post comment on failure
-        if: ${{!success()}}
-        uses: actions/github-script@0.9.0
+        if: ${{failure()}}
```

Comment:
We should keep this as `!success` since jobs can be cancelled and that should post a comment.